### PR TITLE
Fix solr permissions.

### PR DESCRIPTION
**Pull request by busykoala**
Because solr unpacks stuff at startup the permissions on `/parts/solr-instance/solr-webapp/webapp/` are wrong.

### CODE REVIEW

File: production-v2.cfg [hunk 1]
```diff
@@ -13,12 +13,19 @@
 # It is not a replacement for the production.cfg but an extension.
 
 [buildout]
-parts += setpermissions
+parts +=
+    fix-solr-permissions
```
Comment (jone) [inline production-v2.cfg]: Yesterday we have agreed to put that in [`early-parts`](https://github.com/4teamwork/ftw-buildouts/blob/master/production.cfg#L3) so that it is executed _before_ the solr parts.

Why did you change that? Did you gain more insight?

Comment (jone) [reply production-v2.cfg]: Yes it's the correct file.
But you registered the part with `parts +=` and not `early-parts +=`

Comment (busykoala) [reply production-v2.cfg]: Ah, ok. I indeed did not get that part. I changed it now but like to test it before we merge.

File: production-v2.cfg [hunk 2]
```diff
@@ -13,12 +13,17 @@
...
 parts += setpermissions
 
-
 [deployment]
 etc-directory = /apps/etc
 
+[fix-solr-permissions]
+recipe = plone.recipe.command
+command =
+    sudo -u ${buildout:os-user} chmod -R g+w ${buildout:directory}/parts/solr-instance/solr-webapp/webapp
```
Comment (jone) [inline production-v2.cfg]:
```suggestion
    sudo -u ${buildout:os-user} chmod -R --silent g+w ${buildout:directory}/parts/solr-instance/solr-webapp/webapp
```
If no solr is installed, there is no such directory. I suggest to use `--silent` in order to indicate that there is no problem when the directory is missing; we want a `0` exit code anyway.

Comment (busykoala) [reply production-v2.cfg]: Good point. I saw that but thought it might be a good thing to output anyway. But actually we do not need to show this...
